client/constants.js

```diff
@@ -0,0 +1,19 @@
+//const compiledLangs = ["gcc", "gpp", "java"];
+const interpretedLanguages = ["python"];
+const allowedFiles = ["application/zip", "application/x-tar","application/gzip"];
+const allowedExtensions = [".zip", ".tar", ".tar.gz"];
+
+const formId = "mainForm";
+const buildButtonId = "build";
+const runButtonId = "run";
+const outputId = "output";
+const formErrorId = "formError";
+const runCmdId = "runCmd";
+const compileCmdId = "compileCmd";
+const fileId = "file";
+const cmdArgsId =  "cmdArgs";
+
+const getLangHandle = "/getSupportedLanguage";
```

Comment:
maybe getSupportedLanguages?

Reply:
But it is getting only single supported language. On which container code-runner is running